rules-tests/DeadCode/Rector/ClassMethod/RemoveUnusedPrivateMethodRector/Fixture/skip_array_callables_magic_constant_with_constructor.php.inc

```diff
@@ -0,0 +1,27 @@
+<?php
```

Comment:
(note)
This code case shows me unexpected result.

### rectorphp/rector pass
* Skip it.
* https://getrector.com/demo/ceed39e6-2078-42b3-84be-54a0f2035bb6

### rectorphp/rector-src do not pass
* Remove method in use.
<details close>
<summary>PHPUnit result</summary>

```
$ vendor/bin/phpunit -d --enable-pretty-print rules-tests/DeadCode/Rector/ClassMethod/RemoveUnusedPrivateMethodRector/RemoveUnusedPrivateMethodRectorTest.php
PHPUnit 10.5.15 by Sebastian Bergmann and contributors.

Runtime:       PHP 8.3.3
Configuration: /path/to/rector-src/phpunit.xml

....................F..................                           39 / 39 (100%)

Time: 00:01.575, Memory: 90.00 MB

There was 1 failure:

1) Rector\Tests\DeadCode\Rector\ClassMethod\RemoveUnusedPrivateMethodRector\RemoveUnusedPrivateMethodRectorTest::test with data set #20 ('...')
Failed on fixture file "skip_array_callables_magic_constant_with_constructor.php.inc"
Failed asserting that string matches format description.
--- Expected
+++ Actual
@@ @@

         return $array;
     }
-
-    private function sort($a, $b)
-    {
-        return $a <=> $b;
-    }
 }

/path/to/rector-src/src/Testing/PHPUnit/AbstractRectorTestCase.php:248
/path/to/rector-src/src/Testing/PHPUnit/AbstractRectorTestCase.php:164
/path/to/rector-src/rules-tests/DeadCode/Rector/ClassMethod/RemoveUnusedPrivateMethodRector/RemoveUnusedPrivateMethodRectorTest.php:16
```
</details>